django/template/engine.py

```diff
@@ -16,17 +17,26 @@
 
 
 class Engine(object):
+    default_builtins = [
+        'django.template.defaulttags',
+        'django.template.defaultfilters',
+        'django.template.loader_tags',
+    ]
 
     def __init__(self, dirs=None, app_dirs=False,
                  allowed_include_roots=None, context_processors=None,
                  debug=False, loaders=None, string_if_invalid='',
-                 file_charset='utf-8'):
+                 file_charset='utf-8', libraries=None, builtins=None):
+        if builtins is None:
+            builtins = []
```

Comment:
Is there a reason for dealing with `builtins` here? Otherwise, can you keep the order of parameters consistent by moving this just after `libraries`?